### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by six8**

### CODE REVIEW

File: _README.rst [hunk 1]
```diff
@@ -158,33 +158,4 @@ Similar Projects
 * `attrs <https://github.com/hynek/attrs>`_
```
Comment (six8) [inline _README.rst:160]: This file is generated.

File: README.rst [hunk 2]
```diff
@@ -157,39 +157,16 @@ Similar Projects
 * `schematics (formerly dictshield) <https://github.com/j2labs/schematics>`_
 * `attrs <https://github.com/hynek/attrs>`_
 
-
```
Comment (six8) [inline README.rst:160]: This file is generated.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+.PHONY: test
+test: $(SPRINGFIELD_WHEEL)
+	pip install -r test_requirements.txt ./dist/$(SPRINGFIELD_WHEEL)
+	cd src && python tests/runtests.py -v
+
+
+.PHONY: clean
+clean:
+	rm -Rf dist build
```
Comment (BatsShadow) [inline Makefile:51]: nitpick: no trailing newline

**Comment by BatsShadow** [comment]
LGTM

**Pull request merged**